src/docs/goose-book/src/getting-started/validation.md

```diff
@@ -0,0 +1,33 @@
+# Validation
+
+## Goose Eggs
+[Goose-eggs](https://github.com/tag1consulting/goose-eggs) is a validation library that can be used with Goose. 
+
+Include the goose-eggs crate in the dependency section of your `Cargo.toml`. 
+
+```toml
+[dependencies]
+goose-eggs = "0.4.0"
+```
+
+Bring the Validate structure and the validation function that the object is passed to, into the scope of your load test.
+```rust 
+use goose_eggs::{validate_and_load_static_assets, Validate};
+```
+
+We'd probably want to verify the HTTP response status of a request and whether some text appears as we'd expect.
```

Comment:
Now, it is simple to verify that we received a `200` HTTP response status code, and that the text `Gander` appeared somewhere on the page as expected: